<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kết quả chạy</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Thuật toán: FCM&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Đầu ra : Độ đo hiệu năng&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Đầu ra</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>U</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Độ đo hiệu năng</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Kết quả phân cụm</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Thuật toán</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>FCM</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>MC-FCM</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>sSMC-FCM</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_fcm">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QStackedWidget" name="stackWidget_fcm">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="page_fcm_eval">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_fcm_eval">
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_fcm_U">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_fcm_U"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_fcm_V">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_fcm_V"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_fcm_label">
           <layout class="QGridLayout" name="gridLayout_18">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_fcm_label">
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_mc">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QStackedWidget" name="stackWidget_mc">
          <widget class="QWidget" name="page_mc_eval">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_mc_eval">
              <property name="autoScroll">
               <bool>true</bool>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_mc_U">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_mc_U"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_mc_V">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_mc_V"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_mc_label">
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_mc_label"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_ssmc">
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <widget class="QStackedWidget" name="stackWidget_ssmc">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_ssmc_eval">
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_ssmc_eval"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_ssmc_U">
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_ssmc_U"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_ssmc_V">
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_ssmc_V"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_ssmc_label">
           <layout class="QGridLayout" name="gridLayout_16">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_ssmc_label"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Số lần lặp: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
